src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -248,8 +249,12 @@ def _on_focussed_view(self, focussed_view):
         for viewport in self.tab_data_model.viewports.value:
             if viewport.canvas.view.name.value == focussed_view.name.value:
                 viewport.cpol.active.value = True
+                colour = hex_to_frgba(SELECTION_COLOUR)
             else:
                 viewport.cpol.active.value = False
+                colour = (1.0, 1.0, 1.0)  # white
+            if viewport.canvas.bg_view_label:
```

Comment:
I find it odd that the canvas hold a reference to an object part of the text overlay. I think it'd be "cleaner" to not hold that `bg_view_label` explicitly, and here just access it via `canvas.bg_view_overlay.labels[0]` (as done for the `fps_ol`).